.env.example

```diff
@@ -3,4 +3,11 @@ create .env and copy paste following
 DB_NAME = insdb 
 DB_PASSWORD = admin
 DB_USERNAME = postgres
-DATABASE_URL = onheroku
\ No newline at end of file
+DATABASE_URL = onheroku
+Below are secret keys to generate web token 
+You may generate your own secret key by type in the following command
+1. `node`
+2. `require('crypto').randomBytes(64).toString('hex')`
+Note: make sure both secret are different
+ACCESS_TOKEN_SECRET = de9fb3ac2fdd48c5c7549334cb1588065dcfcad3dc94416919cacdeb072de913de4f05a9d933cc5dcdd570158a2d68fa086128fa9184d90078d6ac2604f7931b
+REFRESH_TOKEN_SECRET = e3374c3d9d9196c1efae54c94aba70443c792b8cc39da297dc143ba4d262f1254d9f3e689b357632ff528e502c1595bf3cc084e00eed5acba177fd3d15b9bb82
```

Comment:
token and access token shuoldnt be put here on github, this repo is public, just move it to .env and share the token with the team.